packages/elements/src/pagination/index.ts

```diff
@@ -637,27 +625,23 @@ export class Pagination extends BasicElement {
    * @returns {void}
    */
   private onKeyDown (event: KeyboardEvent): void {
+
     if (event.defaultPrevented) {
       return;
     }
 
-    const newInputValue = Number(this.input.value);
-
+    // Handle keyboard shortcuts
     switch (event.key) {
       case 'Enter':
-        this.updatePageInput();
-        this.input.blur();
-        break;
-      case 'Tab':
-        this.updatePageInput();
+        this.setPage(this.input.value);
```

Comment:
Create a getter for `this.input.value` as well as use https://lit.dev/docs/templates/directives/#ref `ref` directive instead of query.

Reply:
Added